packages/shared/src/forms/SliderMarker.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+import StyleSheet from '../PlatformStyleSheet';
+
+export default function SliderMarker() {
```

Comment:
There is no "pressed" state?

Reply:
Should it be?

Reply:
As a user, I would like some visual feedback that I pressed the handle. But I don't know if the designer has this in mind? I could not find anything in Figma so 🤷‍♂

Reply:
When you have your finger over the marker, you don't see it

Reply:
My finger is not so big :D I guess that's because I'm using the mouse with the emulator.

Reply:
> I guess that's because I'm using the mouse with the emulator

🙈 

I can't see the marker when I put my finger on top of it, but when dragging, it becomes partially visible, but by then, I already know that I have activated it